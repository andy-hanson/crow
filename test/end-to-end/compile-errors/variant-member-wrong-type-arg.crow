main void()
	info log match bar
	as foo
		x
	as bar
		"bar"
	else
		"other"

v[t] variant

foo record string v variant-member
bar record nat v variant-member
