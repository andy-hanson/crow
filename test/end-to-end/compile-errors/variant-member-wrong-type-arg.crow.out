exit code: 0
stdout:
info: bar
stderr:
test/end-to-end/compile-errors/variant-member-wrong-type-arg.crow 3:8-3:11 Type 'foo' is not a member of variant 'nat64 v'.
